.github/licenseList.yaml

```diff
@@ -0,0 +1,211 @@
+licenses:
```

Comment:
This should rather be a map;

```yaml
licenses:
  $image:
    license: $license
    licenseLink: $link
```

This would also prevent accidentally adding an image duplicate

Reply:
This doesn't work.

Map keys must be unique